libs/gi/ui/src/components/artifact/ArtifactCardPico.tsx

```diff
@@ -69,7 +69,6 @@ export function ArtifactCardPico({
             position: 'absolute',
             fontSize: '0.75rem',
             lineHeight: 1,
-            opacity: 0.85,
```

Comment:
I barely even notice this change tbh I was thinking something more like this (just changed backgroundcolor of sqbadge to #00000022)
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/08dee80b-345b-4994-9b45-a74058a339a3)

Reply:
could tweak it later i guess, another option is to apply darker gradients at the topleft and bottom right corner (so there isnt a harsh rectangle line, which is what I'm trying to avoid)